packages/ra-core/src/types.ts

```diff
@@ -278,10 +278,7 @@ export type AdminChildren = RenderResourcesFunction | ReactNode;
 export type TitleComponent = string | ReactElement<any>;
 export type CatchAllComponent = ComponentType<{ title?: TitleComponent }>;
 
-export interface LoginComponentProps {
-    title?: TitleComponent;
-}
-export type LoginComponent = ComponentType<LoginComponentProps>;
+export type LoginComponent = ComponentType<{}> | ReactElement<any>;
 export type DashboardComponent = ComponentType<WithPermissionsChildrenParams>;
```

Comment:
Shouldn't we do the same for the Dashboard ?

Reply:
yes, in due time